apps/ledger-live-desktop/src/renderer/drawers/DataSelector/CurrencyList.jsx

```diff
@@ -45,7 +45,12 @@ export function CurrencyList({ currencies, onCurrencySelect }: Props) {
             {({ index, style }: any) => {
               const currency = currencies[index];
               return (
-                <RowContainer key={index} style={style} onClick={() => onCurrencySelect(currency)}>
+                <RowContainer
+                  key={index}
+                  style={style}
+                  onClick={() => onCurrencySelect(currency)}
+                  data-test-id={`currency-row-${currency.name.toLowerCase()}`}
```

Comment:
is it worth doing `currency?.name?.toLowerCase()`? if you're confident they're always set then ignore this.

Reply:
Yeah probably worth doing. However would the component even load without `currency`?

Reply:
Possibly not. I'm happy for this to go in as is

Reply:
I'll add it and test it. Having it in doesn't do any harm